### PR TITLE
acceptance = backscale x exposure

### PR DESCRIPTION
**Pull request by luca-giunti**
Fix acceptance handling in ogip dataset reader

### CODE REVIEW

**Comment by luca-giunti** [comment]
@registerrier could you merge this please?

File: ogip_spectra/io_ogip.py [hunk 1]
```diff
@@ -212,6 +212,12 @@ def extract_spectrum(pha_table):
             acceptance = pha_meta["BACKSCAL"]
         spectrum_data["acceptance"] = acceptance
 
+        if "EXPOSURE" in pha_table.columns:
```
Comment (registerrier) [inline ogip_spectra/io_ogip.py]: Why would the exposure  be stored in the PHA table?
https://heasarc.gsfc.nasa.gov/docs/heasarc/ofwg/docs/spectra/ogip_92_007/node7.html

**Review by registerrier — COMMENTED** [review]
Thanks @luca-giunti 
I don't understand why the exposure is stored in the pha table. It shouldn't.

**Review by registerrier — APPROVED** [review]
Thanks @luca-giunti

**Pull request merged**